httpx/utils.py

```diff
@@ -84,9 +84,25 @@ def guess_json_utf(data: bytes) -> typing.Optional[str]:
 
 
 def get_netrc_login(host: str) -> typing.Optional[typing.Tuple[str, str, str]]:
-    try:
-        netrc_info = netrc.netrc(os.environ.get("NETRC"))  # type: ignore
-    except FileNotFoundError:
+    if "NETRC" in os.environ:
+        NETRC_FILES = (os.environ["NETRC"],)
+    else:
+        try:
+            NETRC_FILES = (
+                os.path.expanduser("~/.netrc"),
+                os.path.expanduser("~/_netrc"),
+            )
```

Comment:
Rather than doing this every time, lets just populate `NETRC_FILES` once, at the module level.

```python
def get_netrc_files():
    if "NETRC" in os.environ:
         return (os.environ["NETRC"],)
    return (
         os.path.expanduser("~/.netrc"),
         os.path.expanduser("~/_netrc"),
    )

NETRC_FILES = get_netrc_files()
```

Reply:
module level doesn't solve NETRC env changing. change this part could you review again?